tests/test_lsmr.py

```diff
@@ -0,0 +1,43 @@
+import equinox as ex
+import jax.numpy as jnp
+import lineax as lx
+
+
+solver = lx.LSMR(1e-10, 1e-10)
```

Comment:
lineax runs tests with jax in both 32 bit and 64 bit mode so we may want to make these tolerances looser when only using 32 bit